cfgmgr/intfmgr.cpp

```diff
@@ -287,22 +301,149 @@ void IntfMgr::addHostSubIntf(const string&intf, const string &subIntf, const str
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
 }
 
-void IntfMgr::setHostSubIntfMtu(const string &subIntf, const string &mtu)
+
+std::string IntfMgr::getPortAdminStatus(const string &alias)
+{
+    Table *portTable;
+    if (!alias.compare(0, strlen("Eth"), "Eth"))
+    {
+        portTable = &m_statePortTable;
```

Comment:
Not sure if its a good idea to read this each time from the DBs for parent mtu/admin_status. Do you think its better to cache? @venkatmahalingam , @preetham-singh , any thoughts?

Reply:
Are we planning to get the admin_status from cache?

Reply:
Plan to do the caching as another PR